genai/image_generation/imggen_mmflash_locale_aware_with_txt.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package image_generation shows how to use the GenAI SDK to generate images and text.
+package image_generation
+
+// [START googlegenaisdk_imggen_mmflash_locale_aware_with_txt]
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"google.golang.org/genai"
+)
+
+// generateMMFlashLocaleAwareWithText demonstrates how to generate an image with locale awareness.
+func generateMMFlashLocaleAwareWithText(w io.Writer) error {
+	ctx := context.Background()
+
+	client, err := genai.NewClient(ctx, &genai.ClientConfig{
+		HTTPOptions: genai.HTTPOptions{APIVersion: "v1"},
+	})
+	if err != nil {
+		return fmt.Errorf("failed to create genai client: %w", err)
+	}
+
+	modelName := "gemini-2.5-flash-image"
+	prompt := "Generate a photo of a breakfast meal."
```

Comment:
I'm confused about the intent here as the sample file name is called locale_aware and the region name is locale_aware but this prompt is generic. 

What's the intent of the code?